director/scripts/test/JSON_Visualization_README.txt

```diff
@@ -0,0 +1,40 @@
+This documentation explains the different parts of the JSON file needed for the
```

Comment:
Thank you for the excellent documentation. This might be more accessible if it was in `director/scripts.README.md`. All these files also need to be renamed/relocated to a permanent location.